src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/parenpad/InputParenPadToCheckUpperCaseFunctionInOptionProperty.java

```diff
@@ -0,0 +1,33 @@
+/*
+ParenPad
+option = space
```

Comment:
Instead of creating new input file, can we update one of existing.
We do not expect any changes in validation behavior, so content of file does not matter.
So reusage is good.

Reply:
done